### PR TITLE
add-prometheuse-change-desc

### PR DESCRIPTION
**Pull request by qqqdan**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v5.4 (TiDB 5.4 versions)
- [x] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)
- [ ] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Review by srstack — APPROVED** [review]
LGTM

File: releases/release-5.3.0.md [hunk 1]
```diff
@@ -308,6 +308,10 @@ TiDB 在遥测中新增收集 TEMPORARY TABLE 功能的开启情况。收集的
         - 优化监控项和告警规则，提升了同步链路、内存 GC、存量数据扫描过程的可观测性 [#2735](https://github.com/pingcap/tiflow/pull/2735) [#1606](https://github.com/pingcap/tiflow/issues/1606) [#3000](https://github.com/pingcap/tiflow/pull/3000) [#2985](https://github.com/pingcap/tiflow/issues/2985) [#2156](https://github.com/pingcap/tiflow/issues/2156)
         - 当同步任务状态正常时，不再显示历史错误信息，避免误导用户 [#2242](https://github.com/pingcap/tiflow/issues/2242)
 
++ Prometheus
+    - 默认部署 Prometheus [version 2.27.1 ](https://github.com/prometheus/prometheus/releases/tag/v2.27.1) ，是 2021 年发布的较新版本，该版本提供更多的功能并解决了安全风险。相对于 5.3.0 之前版本 TiDB 默认使用的 Prometheus version 2.8.1，存在 Alert 时间格式变化，详情见 [Prometheus commit ](https\://github.com/prometheus/prometheus/commit/7646cbca328278585be15fa615e22f2a50b47d06).
```
Comment (qiancai) [inline releases/release-5.3.0.md:312]: 兼容性变化可以写到前面的  兼容性更改 > 其他  那里

Comment (qqqdan) [reply releases/release-5.3.0.md]: 已修改

File: upgrade-tidb-using-tiup.md [hunk 2]
```diff
@@ -27,6 +27,8 @@ aliases: ['/docs-cn/dev/upgrade-tidb-using-tiup/','/docs-cn/dev/how-to/upgrade/u
     4. 按本文档说明将集群升级到 5.3 版本。
 - 支持 TiDB Binlog，TiCDC，TiFlash 等组件版本的升级。
 - 具体不同版本的兼容性说明，请查看各个版本的 [Release Note](/releases/release-notes.md)。请根据各个版本的 Release Note 的兼容性更改调整集群的配置。
+- 对于从 5.3 之前版本的集群升级到5.3 版本及后续修订版本，默认部署的 Prometheus 会从 v2.8.1 升级到 v2.27.1，新版本提供更多的功能并解决了安全风险。Prometheus v2.27.1 相对于 v2.8.1 存在 Alert 时间格式变化，详情见 [Prometheus commit ](https\://github.com/prometheus/prometheus/commit/7646cbca328278585be15fa615e22f2a50b47d06)。
```
Comment (qiancai) [inline upgrade-tidb-using-tiup.md:30]:
```suggestion
- 对于从 5.3 之前版本的集群升级到 5.3 版本及后续修订版本，默认部署的 Prometheus 会从 v2.8.1 升级到 v2.27.1，新版本提供更多的功能并解决了安全风险。Prometheus v2.27.1 相对于 v2.8.1 存在 Alert 时间格式变化，详情见 [Prometheus commit](https\://github.com/prometheus/prometheus/commit/7646cbca328278585be15fa615e22f2a50b47d06)。
```

**Review by qiancai — APPROVED** [review]

File: releases/release-5.3.0.md [hunk 3]
```diff
@@ -70,6 +70,7 @@ TiDB 版本：5.3.0
 - 修复 v5.3.0 之前的版本中新增索引会导致在规定时间外触发 auto-analyze 的问题。在 v5.3.0 中，用户通过 `tidb_auto_analyze_start_time` 和 `tidb_auto_analyze_end_time` 设定时间段后，只会在该时间段内触发 auto-analyze。
 - plugin 默认存放目录从 `""` 改为 `/data/deploy/plugin`。
 - DM 代码迁移至 [TiCDC 代码仓库的 dm 文件夹](https://github.com/pingcap/tiflow/tree/master/dm)。从 DM v5.3.0 起，DM 采用与 TiDB 相同的版本号。DM v2.0 的下一个主版本为 DM v5.3。DM v2.0 到 v5.3 无兼容性变更，升级过程与正常升级无差异。
+- 默认部署 Prometheus [v2.27.1](https://github.com/prometheus/prometheus/releases/tag/v2.27.1) ，是 2021 年发布的较新版本，该版本提供更多的功能并解决了安全风险。相对于 5.3.0 之前版本 TiDB 默认使用的 Prometheus v2.8.1，v2.27.1 存在 Alert 时间格式变化，详情见 [Prometheus commit](https\://github.com/prometheus/prometheus/commit/7646cbca328278585be15fa615e22f2a50b47d06)。
```
Comment (shichun-0415) [inline releases/release-5.3.0.md:73]:
```suggestion
- 默认部署 Prometheus [v2.27.1](https://github.com/prometheus/prometheus/releases/tag/v2.27.1) ，它是 2021 年发布的较新版本，提供更多的功能并解决了安全风险。相对于 5.3.0 之前版本 TiDB 默认使用的 Prometheus v2.8.1，v2.27.1 存在 Alert 时间格式变化，详情见 [Prometheus commit](https://github.com/prometheus/prometheus/commit/7646cbca328278585be15fa615e22f2a50b47d06)。
```

File: upgrade-tidb-using-tiup.md [hunk 4]
```diff
@@ -27,6 +27,7 @@ aliases: ['/docs-cn/dev/upgrade-tidb-using-tiup/','/docs-cn/dev/how-to/upgrade/u
     4. 按本文档说明将集群升级到 5.3 版本。
 - 支持 TiDB Binlog，TiCDC，TiFlash 等组件版本的升级。
 - 具体不同版本的兼容性说明，请查看各个版本的 [Release Note](/releases/release-notes.md)。请根据各个版本的 Release Note 的兼容性更改调整集群的配置。
+- 对于从 5.3 之前版本的集群升级到 5.3 及后续修订版本，默认部署的 Prometheus 会从 v2.8.1 升级到 v2.27.1，v2.27.1 提供更多的功能并解决了安全风险。Prometheus v2.27.1 相对于 v2.8.1 存在 Alert 时间格式变化，详情见 [Prometheus commit](https\://github.com/prometheus/prometheus/commit/7646cbca328278585be15fa615e22f2a50b47d06)。
```
Comment (shichun-0415) [inline upgrade-tidb-using-tiup.md:30]:
```suggestion
- 升级 v5.3 之前版本的集群到 v5.3 及后续版本时，默认部署的 Prometheus 会从 v2.8.1 升级到 v2.27.1，v2.27.1 提供更多的功能并解决了安全风险。Prometheus v2.27.1 相对于 v2.8.1 存在 Alert 时间格式变化，详情见 [Prometheus commit](https://github.com/prometheus/prometheus/commit/7646cbca328278585be15fa615e22f2a50b47d06)。
```

**Review by shichun-0415 — APPROVED** [review]
LGTM

**Pull request merged**